pkgs/top-level/all-packages.nix

```diff
@@ -30492,7 +30492,7 @@ with pkgs;
 
   kile = callPackage ../applications/editors/kile { };
 
-  kitsas = libsForQt5.callPackage ../applications/office/kitsas { };
+  kitsas = qt6Packages.callPackage ../applications/office/kitsas { };
```

Comment:
I'm not knowledgeable here but I think the preferred pattern these days is to use regular `callPackage` and just depend on `qt6`. You then take the relevant deps out of that package set.

That'd also allow moving this package to by-name.

Reply:
I've made it depend on qt6 instead, moved it to by-name and I've optionally enabled qtwayland for linux.